std/digest/digest.d

```diff
@@ -608,7 +608,7 @@ interface Digest
         /**
          * This is a convenience function to calculate the hash of a value using the OOP API.
          */
-        final @trusted nothrow ubyte[] digest(scope const(void[])[] data...)
+        final @trusted nothrow ubyte[] digest(const(void[])[] data...)
```

Comment:
This one doesn't look wrong. It calls `put` on the individual `void[]`s. `put`'s parameter is `scope`, so it guarantees not to escape the `void[]`. `finish` can't have a reference into `data` then which it could escape.

This hinges on `put`'s `scope` being correct, I guess.